src/main/resources/application.yaml

```diff
@@ -55,3 +55,16 @@ launchdarkly:
     environment: ${ENV_NAME:local}
     user: rd
 
```

Comment:
You'll need to add `  config:
    import: "optional:configtree:/mnt/secrets/rd/"` to `application.yaml` as done in [this PR](https://github.com/hmcts/rd-professional-api/pull/1038/files)

Reply:
done